examples/all-clusters-app/all-clusters-common/src/concentration-measurement-instances.cpp

```diff
@@ -0,0 +1,239 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/clusters/concentration-measurement-server/concentration-measurement-server.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::ConcentrationMeasurement;
+
+static const BitMask<Feature, uint32_t> gConcentrationMeasurementAllFeatures(Feature::kNumericMeasurement,
```

Comment:
- Although `static const` and `constexpr` are essentially the same when it comes to declaring compile-time constants of integer or enum types I think using `constexpr` here would fit better stylistically as its more expressive (and would match the one used in `resource-monitoring-instances.cpp`).
- To also match naming I would suggest to change the name of the feature map
```suggestion
constexpr BitMask<Feature, uint32_t> gConcentrationMeasurementFeatureMap(Feature::kNumericMeasurement,
```